radio/sdcard/horus/WIDGETS/BattCheck/main.lua

```diff
@@ -177,40 +195,46 @@ end
 
 --- Size is 160x30 1/8th
 local function zoneSmall(zone)
-  local myBatt = {["x"]=0, ["y"]=0, ["w"]=75, ["h"]=28, ["segments_w"]=15, ["color"]=WHITE, ["cath_w"]=6, ["cath_h"]=20}
+  local myBatt = {["x"]=0, ["y"]=0, ["w"]=155, ["h"]=35, ["segments_w"]=25, ["color"]=WHITE, ["cath_w"]=6, ["cath_h"]=20}
   local mySensor = getCels(zone.options.Sensor)
+  local isHaveData = (type(mySensor) == "table")
```

Comment:
isHaveDate is not a nice wording, maybe isDataAvailable ?

Reply:
correct, will be changed